configs/ddod/ddod_r50_1x_coco.py

```diff
@@ -0,0 +1,136 @@
+fp16 = dict(loss_scale=512.)
```

Comment:
If you need to use fp16, then the configuration file name should be changed to `ddod_r50_fp16_1x_coco.py`